_posts/2019-06-27-heroku-review-apps-in-multi-tenant-application.markdown

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: "Setup Heroku Review Apps in a multi-tenant application"
+date: 2019-05-17 09:00:00
+categories: ["heroku"]
+author: "cleiviane"
+---
+
+Recently at [Ombulabs](https://www.ombulabs.com) I had the mission of setting up Heroku [Review Apps](https://devcenter.heroku.com/articles/github-integration-review-apps) for one of our projects. Since the project is built in a multi-tenant architecture (using the [apartment gem](https://github.com/influitive/apartment)) it was a little challengeable to setup.
```

Comment:
```suggestion
Recently at [Ombu Labs](https://www.ombulabs.com) I had the mission of setting up Heroku [Review Apps](https://devcenter.heroku.com/articles/github-integration-review-apps) for one of our projects. Since the project is built in a multi-tenant architecture (using the [apartment gem](https://github.com/influitive/apartment)) it was a little challengeable to setup.
```